transactions/set-version-boundary/2025/may-15/arguments.json

```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "type": "UInt8",
+    "value": "0"
+  },
+  {
+    "type": "UInt8",
+    "value": "41"
+  },
+  {
+    "type": "UInt8",
+    "value": "0"
+  },
+  {
+    "type": "String",
+    "value": ""
```

Comment:
@janezpodhostnik / @j1010001  is this correct? there is no `rc-1` for this build so I am leaving it as an empty string.

Reply:
yes, no patch, but it should be "null" string I think, see: https://www.notion.so/flowfoundation/Testnet52-Height-Coordinated-Upgrade-10-19e1aee1232480d58423dffdd0b5151b?pvs=4#19e1aee1232481b0a2f1eeb832cf204e

Reply:
this should be:
```json
 {
    "type": "Optional",
    "value": null
  },
```

Reply:
see: https://github.com/onflow/service-account/pull/264/files#diff-6cccc1c2d275510ce056607bac813980ca6ead6bbe28703e8128078c3efa0b0b

Reply:
thanks! updated to `null`